### ISSUE TITLE
Hass Group Helper

### ISSUE THREAD

**Issue by MomosX** [hasscontrol/hasscontrol#70]
Since some versions back, HomeAssitant has a group helper.   I created a switch group helper named garmin that contains 3 entities. 
ID of the helper/group is switch.garmin

However if i enter this into settings and then refresh the entities i get "Resource not found".  
I think the problem comes from the fact that in settings one needs to enter "group" in front.   I tried any combo possible , no luck.

Anyone has any idea please ?

Thank you!

**Comment by medoix** [comment]
hasscontrol uses the [old style groups](https://www.home-assistant.io/integrations/group/#old-style-groups) and these can only be created via the YML config file. All groups created via the HASS UI are now new groups that start with switch. lock. media_player. etc

**Comment by MomosX** [comment]
Thank you.  That i know.  Question is if it can be changed to accomodate the new HAS groups.  Trying to move away from yaml, not back to it :)

**Comment by BebeMischa** [comment]
No changes, please, only ADDITIONS.
I understand the need for this, fully support the idea too, but not as a change, but as an addition. Not everyone is leaving yaml behind and not everyone would be happy, if his working system fails because of an update... ;-)

**Comment by MomosX** [comment]
Agreed on only additions obviously. What i was saying is if the new HAS groups could be accomodated it would be great.  I would like very much to use this but i am trying to move away from YAML for quite some time now, as this seems also the way HAS is going for the future.
